Source/WebCore/platform/graphics/FloatPoint.h

```diff
@@ -26,6 +26,7 @@
 
 #pragma once
 
+#include "AffineTransform.h"
```

Comment:
Usually I try to get this working without modifying the WebKit and WebCore headers.
My solution is to move WebKit and webcore includes to local includes in the CMakeLists (you can see how this is done in PlatformHaiku.cmake for WebKitLegacy).
This way, the include with quotes will include WebKit header, and the include with brackets will include Haiku headers

Reply:
@pulkomandy ur saying about local include directories?

Reply:
@pulkomandy i did this because to avoid the forward declaration problem that was stopping the build!

Reply:
In this case it is referring to the WebCore header, but if you do nothing special, #include "AffineTransform.h" will include the Haiku header instead, and the build will be confused.